bundle/config/resources/mlflow_model.go

```diff
@@ -0,0 +1,7 @@
+package resources
+
+import "github.com/databricks/databricks-sdk-go/service/mlflow"
+
+type MlflowModel struct {
+	*mlflow.RegisteredModel
```

Comment:
Question - any reason this uses Model and not CreateModelRequest?

Reply:
https://github.com/databricks/cli/pull/2828

Reply:
I don't recall...

I thought it had to do with tags, but now I see the struct for create also includes tags, so that can't be it.